Gemfile

```diff
@@ -35,6 +35,16 @@ gem 'rails_12factor', group: :production
 group :development do
   gem 'letter_opener'
   gem 'quiet_assets' # mutes asset pipeline log messages
+  gem 'better_errors'
```

Comment:
Gem 'better errors' was for our own local machines to help with the errors. 
Gem 'letter opener' is to test the emails sent by Speakerinnen to anyone who registers
We added gem guard to help with automatic testing